### PR TITLE
Update 06. blocks_and_widgets.md

### PR DESCRIPTION
**Pull request by kpieters**
## Type

- Enhancement

## Pull request description

Documentation fix
Added return types and double closing ) in the parse function

### CODE REVIEW

File: docs/05. module guide/06. blocks_and_widgets.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ private $record;
 Then we define our (private) variables. In our case we'll use an array to save the record with the article we will be viewing.
 
 ```
-public function execute()
+public function execute(): void
 {
  // call the parent
```
Comment (jeroendesloovere) [inline docs/05. module guide/06. blocks_and_widgets.md]: The indentation of the entire php code block does not seem valid.

Comment (kpieters) [reply docs/05. module guide/06. blocks_and_widgets.md]: I just changed is online and it's the same indent as in the core.,
Only added the :void

Comment (kpieters) [reply docs/05. module guide/06. blocks_and_widgets.md]: My mistake, forgot to download the example module and just checked the code with the documentation by copy pasting it and creating a new MiniBlog module by myself.

Comment (jeroendesloovere) [reply docs/05. module guide/06. blocks_and_widgets.md]: Never mind, we will change this after your PR

**Comment by carakas** [comment]
@StijnVrolijk can you check again?

**Pull request merged**